manifests/README.md

```diff
@@ -1,32 +1,55 @@
-# Dashboard
+# Manifests
 
-The Open Data Hub Dashboard component installs a UI which 
+The Dashboard manifests run on Kustomize. There are 3 types of deployments for the Dashboard component.
 
-- Shows what's installed
-- Show's what's available for installation
-- Links to component UIs
-- Links to component documentation
+- Open Data Hub ([`./odh`](./odh))
+- Red Hat OpenShift AI
+  - RHOAI Managed ([`./rhoai/addon`](./rhoai/addon))
+  - RHOAI Self Managed ([`./rhoai/onprem`](./rhoai/onprem))
 
-For more information, visit the project [GitHub repo](https://github.com/opendatahub-io/odh-dashboard).
+## Adding/Modifying Manifests
 
-### Folders
-1. base: contains all the necessary yaml files to install the dashboard
+Rules for keeping the manifest files in a sane order:
 
-##### Installation
-Use the `kustomize` tool to process the manifest for the `oc apply` command.
+1. When adding a new type of thing, always make it have its own folder; be sure to add the reference to the parent folder's `kustomziation.yaml` (if applicable) 
+2. When adding to a preexisting folder, be sure to add it to the root `kustomization.yaml` in that folder
+3. Do not reference "a file" (has an extension) inside another folder. Reference other folders, which will pick up the `kustomization.yaml` inside that folder; those `kustomization.yaml` files should reference sibling files
+4. Folders matter -- see the README in each for more details
 
+## Installation (ODH)
+
+You can use the `kustomize` tool to process the manifest for the `oc apply` command.
+
+```markdown
+# Set the namespace in the manifest where you want to deploy the dashboard
+kustomize edit set namespace <DESTINATION NAMESPACE>
+kustomize build common | oc apply -f -
+kustomize build overrides/base | oc apply -f -
```

Comment:
Now that we've changed the folders, we had some scripts under `install` (`deploy.sh`, `undeploy.sh`), to automate the development section, we can either remove them or adapt them to this new reestructure.